Integrations/integration-HybridAnalysis.yml

```diff
@@ -354,6 +363,186 @@ script:
 
     }
 
+
+    function quick_scan_result_to_entry(response, indicator, threshold, headers) {
+
+        var unknown_counter = 0 ;
+        var malicious_counter = 0 ;
+        //var num_elements = 0 ;
+        var Score = 0;
+
+        // create scanners entry
+        var scanners = [];
+        response.scanners.forEach(function(key) {
+            scanners.push(
+                {
+                    'Name': key.name,
+                    'Status': key.status,
+                    'Positives': key.positives
+                });
+            if (key.status == 'unknown' || key.status =='no-result' )
+                unknown_counter++;
+            else if (key.status == 'malicious')
+                malicious_counter++;
+       //     num_elements++;
+        });
+
+        //calculating DBot Score
+        if (unknown_counter == response.scanners.lenght)
+            Score = 0;
+        if (malicious_counter == 0)
+            Score = 1;
+        if (malicious_counter == 1)
+            Score = 2;
+        if (malicious_counter >= threshold)
+            Score = 3;
+
+        var Dbot_context = {
+            'Indicator': response.sha256,
+            'Vendor': "Hybrid Analysis",
+            'Score': Score,
+            }
+        var whitelist = {
+            'ID': response.whitelist[0].id,
+            'Value' : response.whitelist[0].value
+        }
+        var res = {};
+        if (Score == 3){
+            res.Malicious = {
+                Vendor: 'Hybrid Analysis',
+                Description: 'DBotScore is 3' // check this!
+            };
+        }
+
+        //create context
+        var file_context = {
+            'ID': response.id,
+            'SHA256': response.sha256,
+            'Scanner' : scanners,
+            'Finished': response.finished,
+            'Whitelist': whitelist
+        };
+
+        if (indicator == 'File') {
+            res.EntryID = response.id;
+            res.SHA256 = response.sha256;
+            Dbot_context.Type = 'File';
+            var context= {
+                'HybridAnalysis.File(val.id && val.id == obj.id)' : file_context,
+                'File(val.id && val.id == obj.id)' : res,
+                'DBotScore' : Dbot_context
+            };
+        }
+
+        else {
+            Dbot_context.Type = 'URL';
+            var context= {
+                'HybridAnalysis.URL(val.id && val.id == obj.id)' : file_context,
+                'DBotScore' : Dbot_context
+            }
+            // if (Score == 3)
+            //     context.URL.Malicious = res;
+        }
```

Comment:
I think that a good coding practice is to try to keep functions short.
For example, this function is about 80 lines long - making it very difficult to see it's purpose in a quick look.
If we export smaller purposes to other functions we could make this function shorter and clearer